python/cufinufft/tests/test_error_checks.py

```diff
@@ -64,6 +64,54 @@ def test_set_pts_raises_on_size():
     assert '`z` must be of shape' in err.value.args[0]
 
 
+def test_set_pts_raises_on_nonvector():
```

Comment:
I like these analogs of "dumpinputs" for CPU tests.

Reply:
Yeah I want to make sure that as many lines of code as possible are tested (which includes checking errors, of course).